core/src/test/java/org/elasticsearch/search/aggregations/InternalAggregationTestCase.java

```diff
@@ -122,4 +136,49 @@ protected final T createUnmappedInstance(String name) {
     protected NamedWriteableRegistry getNamedWriteableRegistry() {
         return namedWriteableRegistry;
     }
+
+    public void testFromXContent() throws IOException {
```

Comment:
shall we make it final then and remove InternalCardinalityTests#testFromXContent ?